pavelib/utils/test/suites/nose_suite.py

```diff
@@ -128,7 +128,7 @@ def __init__(self, *args, **kwargs):
         if self.processes is None:
             # Use one process per core for LMS tests, and no multiprocessing
             # otherwise.
-            self.processes = 0
+            self.processes = -1 if self.root == 'lms' else 0
```

Comment:
@cpennington Is there an easy way to disable this option when running tests locally? I've found that `from nose.tools import set_trace; set_trace()` fails to stop when I'm attempting to debug a test, given this change. Reverting this line seems to fix things.

https://gist.github.com/efischer19/c951a88991ce0fa09b2afb7eb2fa18b3